tests/screenshot/apps/components/Layout.js

```diff
@@ -41,7 +41,25 @@ const rowTestCases = [
 		<Cell className={css.blue} shrink>Layout menu</Cell>
 		<Cell className={css.red}>Layout Content</Cell>
 		<Cell className={css.blue} shrink>Layout Side</Cell>
-	</Layout>
+	</Layout>,
+
+	<Row>
+		<Cell className={css.red} grow>Layout Head</Cell>
+		<Cell className={css.blue} grow>Layout Content</Cell>
+		<Cell className={css.green} grow>Layout Tail</Cell>
+	</Row>,
+
+	<Row>
+		<Cell className={css.red} grow size={1000}>Layout Head</Cell>
```

Comment:
The above Row result and this Row result look the same. Is this intended? Or should we change the size..?

Reply:
This is intended. The result of both is the same. However, I want to test that when the sum of the sizes of three Cells is smaller than the size of the container, the three Cells are arranged side by side and fill the remaining space of the container.